biogrid/libs/biogrid-simulator/src/lib/biogrid/biogrid.spec.ts

```diff
@@ -22,7 +24,7 @@ beforeAll(() => {
     new Building(BUILDING.DEFAULT_INITIAL_ENERGY, 9, 9, name5),
   ];
   grid = new Biogrid(
-    new RuralArea(ruralArea, /* townWidth = */ 10, /* townHeight = */ 10),
+    new RuralArea(ruralArea, /* townWidth = */ townWidth, /* townHeight = */ townHeight),
```

Comment:
now that you have variable names for townWidth and townHeight, you can get rid of the inline comments